spec/intl.html

```diff
@@ -1676,20 +1676,63 @@ <h1>
           </dl>
         </emu-clause>
 
-        <emu-clause id="sec-temporal-calendardatemergefields" type="abstract operation">
+        <emu-clause id="sec-temporal-calendarmutuallyexclusivekeys" type="abstract operation">
           <h1>
-            CalendarDateMergeFields (
+            CalendarMutuallyExclusiveKeys (
               _calendar_: a String,
-              _fields_: an Object,
-              _additionalFields_: an Object,
-            ): an Object
+              _key_: a property key,
+            ): a List of property keys
           </h1>
           <dl class="header">
             <dt>description</dt>
-            <dd>It takes two Objects of calendar-specific fields for the calendar represented by _calendar_ in _fields_ and _additionalFields_ and returns a new Object that includes both sets of fields. The values in _additionalFields_ should supersede the values in _fields_. Also, the returned Object must be free of ambiguity or conflicts. This is relevant for calendars which accept fields other than the standard set of built-in calendar fields.</dd>
+            <dd>
+              It performs the calendar-specific operation of determining which fields are mutually exclusive for the field given by _key_.
+              The returned List may be empty.
+            </dd>
           </dl>
+          <p>
+            Mutually exclusive means that the fields are alternate, potentially conflicting, ways of expressing the same data.
+            This operation is relevant for calendars which accept fields other than the standard set of ISO calendar fields, in order to implement the Temporal objects' `with()` methods, and `Temporal.Calendar.prototype.mergeFields()` in such a way that the result is free of ambiguity or conflicts.
+          </p>
+          <p>
+            For example, given a _calendar_ that uses eras, such as *"gregory"*, a _key_ of *"year"* would mutually exclude *"era"* and *"eraYear"*, and vice versa, because specifying *"era"* and *"eraYear"* is an alternate way of specifying *"year"* and the two could potentially conflict.
+            In addition to the ISO 8601 mutual exclusion of *"month"* and *"monthCode"*, a possible implementation might produce the following results when _calendar_ is *"gregory"*:
+          </p>
+          <emu-table id="table-calendarmutuallyexclusivekeys-example">
+            <emu-caption>Example results of CalendarMutuallyExclusiveKeys</emu-caption>
+            <table>
+              <thead>
+                <tr>
+                  <th>_key_</th>
+                  <th>Returned List</th>
+                </tr>
+              </thead>
+              <tbody>
+                <tr>
+                  <td>*"era"*</td>
+                  <td>« *"year*" »</td>
```

Comment:
As per https://github.com/tc39/proposal-temporal/issues/2465#issuecomment-1402211524 we need to exclude eraYear here too.